morphir/runtime/src/org/finos/morphir/runtime/Runtime.scala

```diff
@@ -1,14 +1,12 @@
 package org.finos.morphir
 package runtime
 
+import org.finos.morphir.ir.Type.UType
 import org.finos.morphir.naming._
 trait Runtime { self =>
   import Runtime._
 
-  type TypeAttribs
```

Comment:
Similar to the above, these are unrelated to `TypedMorphirRuntimeDefs` and should not have been changed.

Reply:
(I see a few places where that happened - anywhere where `TypeAttribs` and `ValueAttribs` are defined locally rather than imported from `TypedMorphirRuntimeDefs` we should leave be.)